lib/external/sentry.js

```diff
@@ -91,6 +91,10 @@ const sanitizeEventRequest = (event) => {
       // filter out access tokens embedded in URLs
       event.request.url = filterTokenFromUrl(event.request.url);
     }
+
+    event.extra = {
+      duration: `${Math.round((Date.now() - event.request.headers['x-request-received-at']) / 1000)} sec`
```

Comment:
Is rounding this useful?  I'd expect milliseconds to be a more useful resolution here.  Hopefully most requests would `round()` to 0 seconds :grin: